org.gtk.Gtk3theme.SolArc-Dark.json

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "org.gtk.Gtk3theme.SolArc-Dark",
+  "branch": "3.22",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
```

Comment:
Any reason for not using 18.08?  I have no idea if the Gtk themes do work as expected with 18.08.